.github/workflows/model_servers.yaml

```diff
@@ -97,7 +99,7 @@ jobs:
         with:
           image: ${{ env.REGISTRY }}/${{ github.repository_owner}}/${{ matrix.image_name }}
           platforms: ${{ matrix.platforms }}
-          tags: latest
+          tags: latest ${{ github.ref_type == 'tag' && github.ref_name || '' }}
```

Comment:
Does this end up with 
tags: latest githup.ref_name? if set?

Or should it be just 

tags: githup.ref_name

Reply:
`tags: latest githup.ref_name? if set?` yes